ui/app/pages/settings/settings.container.js

```diff
@@ -50,33 +46,24 @@ const mapStateToProps = (state, ownProps) => {
   const isMyAccountsPage = pathname.match('my-accounts');
```

Comment:
This variable can be removed, since there are no longer any matching paths. Any conditions dependent upon this being true can be removed as well.